extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/MSQTestBase.java

```diff
@@ -251,6 +251,7 @@ public class MSQTestBase extends BaseCalciteQueryTest
                   .put(MultiStageQueryContext.CTX_MAX_NUM_TASKS, 2)
                   .put(MSQWarnings.CTX_MAX_PARSE_EXCEPTIONS_ALLOWED, 0)
                   .put(MSQTaskQueryMaker.USER_KEY, "allowAll")
+                  .put("enableWindowing", true)
```

Comment:
I thought we removed that Context no ?
Also we should put : https://github.com/apache/druid/blob/5c7512c54db85f1db6e550777a6cdad04bd5a10e/sql/src/main/java/org/apache/druid/sql/calcite/planner/PlannerContext.java#L90

Reply:
Context is removed and the `CTX_ENABLE_WINDOW_FNS` is removed too